src/components/SignUp/SignUpFooter.tsx

```diff
@@ -92,8 +92,8 @@ const SignUpFooter: React.FC<{
                 ? privacyAgree
                 : () => setLandingStep(2)
             }
+            style={{ background: colors.darkGreen, border: colors.darkGreen }}
             type="button"
-            variant="primary"
```

Comment:
Don't remove the variant here, too. You probably don't need to make any changes to the footer file for this ticket